crates/node/src/interactive_cli/context.rs

```diff
@@ -640,3 +676,96 @@ fn handle_alias_command(node: &Node, command: AliasCommands, ind: &str) -> EyreR
 
     Ok(())
 }
+
+async fn handle_proposals_command(
+    node: &Node,
+    command: ProposalsCommands,
+    ind: &str,
+) -> EyreResult<()> {
+    match command {
+        ProposalsCommands::List {
+            context,
+            offset,
+            limit,
+        } => {
+            let context_id = node
+                .ctx_manager
+                .resolve_alias(context, None)?
+                .ok_or_eyre("unable to resolve context")?;
+
+            // Use proposal_manager instead of ctx_manager
+            let proposals = node
+                .proposal_manager
+                .get_proposals(&context_id, offset, limit)
+                .await?;
+
+            if proposals.is_empty() {
+                println!("{ind} No proposals found for context '{}'", context_id);
+            } else {
+                println!("{ind} Proposals for context '{}':", context_id);
+                for proposal in proposals {
+                    println!(
+                        "{ind} - Proposal ID: {}, Author: {}",
+                        format!("{:?}", proposal.id).cyan(),
```

Comment:
won't this print `Repr { inner: ProposalId(Identity([ <32 bytes> ] )) }`

make a `Hash` instead, and print that

`Hash::from(proposal.id.as_bytes())`